crates/sncast/src/response/structs.rs

```diff
@@ -1,7 +1,7 @@
 use camino::Utf8PathBuf;
 use conversions::serde::serialize::CairoSerialize;
 use serde::{Deserialize, Serialize, Serializer};
-use starknet::core::types::FieldElement;
+use starknet::core::types::{FieldElement, TransactionStatus};
```

Comment:
Please create an internal structure instead. This defines the command output, this way library bump can unintentionally change what the command prints.